app/api/sessions.py

```diff
@@ -194,11 +197,32 @@ def before_get_object(self, view_kwargs):
             event = safe_query(self, Event, 'identifier', view_kwargs['event_identifier'], 'identifier')
             view_kwargs['event_id'] = event.id
 
+    def after_update_object(self, session, data, view_kwargs):
+        """ Send email if session accepted or rejected """
+        if session.state == 'accepted' or session.state == 'rejected':
```

Comment:
Not sure, but wouldn't this send mail every time the resource is updated and the session is either accepted or rejected ?